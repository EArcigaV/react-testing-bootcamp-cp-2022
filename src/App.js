import {
  MainContainer,
  HeaderClass,
  ContentClass,
  FooterClass,
} from "./App.styles";

import "./App.css";
import Header from "./components/Header/Header";
import Content from "./components/Content/Content";
//import DayPicker from "./components/Content/DayPicker/DayPicker";
import Footer from "./components/Footer/Footer";
//import BasicDatePicker from "./basicDatePicker";

function App() {
  return (
    <MainContainer>
      <HeaderClass>
        <Header />
      </HeaderClass>
      <ContentClass>
        <Content />
        {/* <BasicDatePicker /> */}
      </ContentClass>
      <FooterClass>
        <Footer />
      </FooterClass>
    </MainContainer>
  );
}

export default App;
