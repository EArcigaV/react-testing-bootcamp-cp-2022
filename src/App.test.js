import { render, screen } from "@testing-library/react";

import App from "./App";

test("renders App correctly ", () => {
  const { container } = render(<App />);
  expect(container).toMatchSnapshot();
});
