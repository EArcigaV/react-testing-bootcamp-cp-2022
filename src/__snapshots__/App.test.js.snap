// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App correctly  1`] = `
<div>
  <div
    class="sc-bczRLJ laIWnM"
  >
    <div
      class="sc-gsnTZi dhBUzX"
    >
      <h3>
        My testing-React Project 2022
      </h3>
    </div>
    <div
      class="sc-dkzDqf clmMnX"
    >
      <h2>
        Loading...
      </h2>
    </div>
    <div
      class="sc-hKMtZM eyVwxs"
    >
      <div>
        Project created during Wizeline Academy React Testing Bootcamp
      </div>
    </div>
  </div>
</div>
`;
